src/HumanView.cpp

```diff
@@ -78,6 +236,34 @@ void HumanView::drawEndLevel() {
     display.display();
 }
 
+void HumanView::drawFinalScore() {
+    display.clear();
+    sf::Text score;
+    score.setFillColor(sf::Color::Red);
+    score.setCharacterSize(20);
+    score.setFont(font);
+    float posY = height / 8;
+    for (int x = 0; x < 9; x++) {
+        score.setString(strings.getLevelScoreString(x, logic.getScore(x)));
+        score.setPosition(width / 6, posY);
+        display.draw(score);
+        posY += 30;
+    }
+    score.setString(strings.getString("Tier") + strings.getTier((levelsWon-1)/2));
+    score.setPosition(width / 6, posY + 30);
+    display.draw(score);
+    score.setString(strings.getString("FinalInstruct"));
+    score.setPosition(width / 6, posY + 60);
+    display.draw(score);
+    display.display();
+}
+
+void HumanView::checkKeyboardFinal() {
```

Comment:
Add option to go back to menu screen or continue to next level when the player reaches the score screen after each level